resources/views/components/user/game-activity.blade.php

```diff
@@ -47,6 +47,18 @@
                                     @endif
                                     </span>
                                 @endif
+                                @if ($session['playerSession'] && $session['playerSession']['gameHash'])
+                                    @if ($session['userAgent'])
+                                        <span class="text-muted"> | </span>
+                                    @endif
+                                    @php $hash = $session['playerSession']['gameHash']; @endphp
+                                    <span class="smalltext" title="{{ $hash['name'] }}">
+                                        {{ $hash['md5'] }}
```

Comment:
Looks good - only thing I'm wondering is if it may be worth flipping this (put the md5 in the tooltip and have the name visible). Users probably will just see a blur of characters for the md5, but I would imagine the hash name is much more recognizable.

Reply:
I considered that, but decided against it for reasons that I've already forgotten. I'll switch it around.

I've also changed the order so the hash description appears first:
![image](https://github.com/user-attachments/assets/5a4ac206-4752-4c9d-aba5-943e278d8351)
![image](https://github.com/user-attachments/assets/36af8135-2582-400f-9080-8cb19b035c77)